tasks/48_follow-the-spy/follow-the-spy.js

```diff
@@ -0,0 +1,20 @@
+export function followTheSpy(arr) {
+  let result = '';
+  result = result.concat(arr[0][0] + ', ' + arr[0][1]);
+  let j = 0;
```

Comment:
it looks not very clear to understand what is going on with this j and loop which can only break at certain condition, you can take use a find method to solve this in more readable way